docs/source/reference/artifacts.rst

```diff
@@ -4,7 +4,12 @@ optuna.artifacts
 ================
 
 The :mod:`~optuna.artifacts` module provides the way to manage artifacts (output files) in Optuna.
-Please note that methods defined in each ArtifactStore are not intended to be directly accessed by library users.
+
+.. note::
+   The methods defined in each ``ArtifactStore`` are not intended to be directly accessed by library users.
+
+.. note::
+   As ``ArtifactStore`` does not officially provide user API for artifact removal, please refer to `FAQ <how-can-i-delete-all-the-artifacts-uploaded-to-a-study>`_ for the hack.
```

Comment:
The link to the FAQ is broken. You need to insert the custom anchor to the FAQ page. Could you fix it?

* https://optuna--5501.org.readthedocs.build/en/5501/reference/artifacts.html
* https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor

Reply:
I addressed your suggestion!
Let me see how the test goes!